### DISCUSSION TITLE
Bastion host key has changed!

### DISCUSSION THREAD

**Discussion by dmtri35** [WATonomous/infrastructure-support#23] [General Announcements]
Hi @WATonomous/compute-cluster-users,

During the `bastion` outage yesterday, we have removed and regenerated the public key for `bastion` so if you login to the cluster now, you might see a warning saying *host key has changed*. You can safely ignore this warning.

Thanks for your patience,
Tri

**Comment by mxpoch** [comment]
Hi Tri,

Even after adding a "-o StrictHostKeyChecking=no" option to my ssh command, the shell panics and tells me to update the host key in  /Users/max/.ssh/known_hosts.

What can I do to get connected?

**Reply by dmtri35** [reply]
Hi Max,

You should follow the instructions given by the shell. To update the host in known_hosts, you have two options:

1. Open the known_hosts file and delete the bastion entry manually
2. Run the command suggested by the shell.

Then when you try reconnecting to bastion, it would be the same as if you are connecting to bastion for the first time.

Hope that helps!
Tri

**Reply by mxpoch** [reply]
First option worked like a charm, thanks Tri!
